.github/workflows/playground.yml

```diff
@@ -34,10 +35,24 @@ jobs:
           pnpm install --frozen-lockfile
           pnpm build
           pnpm playground
-      - name: Deploy 🚀
+      - name: Deploy Main 🚀
+        if: github.ref == 'refs/heads/main'
```

Comment:
We need a `main` directory here, see above.

Reply:
I'll also update the README to point to the new location then as well.